packages/prover/src/verified_requests/eth_getTransactionCount.ts

```diff
@@ -0,0 +1,30 @@
+import {ELVerifiedRequestHandler} from "../interfaces.js";
+import {bufferToHex} from "../utils/conversion.js";
+import {getELProof, isValidAccount, isValidStorageKeys} from "../utils/execution.js";
+import {generateRPCResponseForPayload, generateUnverifiedResponseForPayload} from "../utils/json_rpc.js";
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export const eth_getTransactionCount: ELVerifiedRequestHandler<
```

Comment:
@nazarhussain since it shares most of the steps from `ethGetBalance`, can we refactor to use a common method? could be in a separate PR